### PR TITLE
Fix 1172

### PR DESCRIPTION
**Pull request by tranCzhu**

### CODE REVIEW

File: cards/src/main/resources/cards/custom/group3/all_hallows_end/minion_wee_squashling.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "targetSelection": "ANY",
     "spell": {
       "class": "HealSpell",
-      "value": 4
+      "value": 2
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/all_hallows_end/minion_wee_squashling.json]: Is this supposed to be changed?

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_dragon_caretaker.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Dragon Caretaker",
+  "baseManaCost": 1,
+  "type": "MINION",
+  "heroClass": "RUST",
+  "baseAttack": 2,
+  "baseHp": 1,
+  "rarity": "RARE",
+  "description": "Battlecry: Give a friendly Dragon +2 Health.",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_dragon_caretaker.json]: Use Opener instead of Battlecry

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_hydraquatom.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Hydraquatom",
+  "baseManaCost": 8,
+  "type": "MINION",
+  "heroClass": "RUST",
+  "race": "MECH",
+  "baseAttack": 8,
+  "baseHp": 8,
+  "rarity": "LEGENDARY",
+  "description": "Deathrattle: Fire four missiles at random enemies that deals 4 damage each.",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_hydraquatom.json]: `Aftermath: Fire four missiles at random enemies that deal 4 damage each.`

^deathrattle to aftermath, deals to deal

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_ethereal_knight.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "target": "SELF",
+    "value": {
+      "class": "EntityCounter",
+      "target": "FRIENDLY_MINIONS"
+    },
+    "attribute": "AURA_ATTACK_BONUS"
+  },
+  "attributes": {},
+  "collectible": true,
+  "set": "RISE_OF_SHADOWS",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_ethereal_knight.json]: Should be CUSTOM set

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_hydraquatom.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "class": "MissilesSpell",
+    "target": "ENEMY_CHARACTERS",
+    "value": 4,
+    "howMany": 4
+  },
+  "attributes": {
+    "DEATHRATTLES": true
+  },
+  "collectible": true,
+  "set": "RISE_OF_SHADOWS",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_hydraquatom.json]: Should be CUSTOM set

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_irena_dragon_knight.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+        "class": "SetAttributeSpell",
+        "value": 2,
+        "attribute": "AURA_ATTACK_MULTIPLIER"
+      },
+      "removeEffect": {
+        "class": "RemoveAttributeSpell",
+        "attribute": "AURA_ATTACK_MULTIPLIER"
+      }
+    },
+"attributes": {
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_irena_dragon_knight.json]: Format this file

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_irena_dragon_knight.json [hunk 7]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Dragon Caretaker",
+  "baseManaCost": 3,
+  "type": "MINION",
+  "heroClass": "RUST",
+  "baseAttack": 2,
+  "baseHp": 1,
+  "rarity": "LEGENDARY",
+  "description": "Blitz. Adjacent Dragons have double their Attack.",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_irena_dragon_knight.json]: Change to `Blitz. Adjacent Dragons have double Attack.`

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_irena_dragon_knight.json [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+  "aura":
+    {
+      "class": "Aura",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_irena_dragon_knight.json]: This should probably be a `StatefulAttributeValueAura`. Use the `AURA_ATTACK_MULTIPLIER` as the attribute, and remove the apply/remove effect and remove always apply.

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_blastflame_dragon.json [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+        },
+        "value": 7,
+        "randomTarget": true
+      }
+    }
+  },
+  "attributes": {
+    "SUPREMACY": true
+  },
+  "collectible": false,
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_blastflame_dragon.json]: This should be `"collectible": true`

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_scarlet_dragon.json [hunk 10]
```diff
@@ -9,21 +9,21 @@
   "race": "DRAGON",
   "description": "Supremacy: Restore this minion to full Health.",
   "trigger": {
-    "eventTrigger": {
-      "class": "SupremacyTrigger"
-    },
+  "eventTrigger": {
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_scarlet_dragon.json]: Format this file.

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_tempest_sentinel.json [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+  "description": "Your spells cast on minions cost (1) less.",
+  "cardCostModifier": {
+    "class": "CardCostModifier",
+    "value": 1,
+    "cardType": "SPELL",
+    "operation": "SUBTRACT",
+    "targetPlayer": "BOTH"
+  },
+  "collectible": true,
+  "set": "BLIZZARD_ADVENTURE",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_tempest_sentinel.json]: this should be custom

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_tempest_sentinel.json [hunk 12]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Tempest Sentinel",
+  "baseManaCost": 5,
+  "type": "MINION",
+  "heroClass": "RUST",
+  "baseAttack": 4,
+  "baseHp": 5,
+  "rarity": "RARE",
+  "race": "ELEMENTAL",
+  "description": "Your spells cast on minions cost (1) less.",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_tempest_sentinel.json]: This doesn't quite do stuff only for minions yet, so we have to revisit it

File: cards/src/main/resources/cards/custom/group3/set_1087/minion_tower_ophidian.json [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+  "description": "Supremacy: Summon a copy of this minion.",
+  "trigger": {
+    "eventTrigger": {
+      "class": "SupremacyTrigger"
+    },
+    "spell": {
+      "class": "SupremacySpell",
+      "spell": {
+        "class": "SummonSpell",
+        "card": "minion_tower_ophidian"
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/minion_tower_ophidian.json]: this should be `"target": "SELF"` instead of `"card...` because it's summoning a copy of itself, not another copy of a Tower Ophidian generally.

File: cards/src/main/resources/cards/custom/group3/set_1087/spell_dawns_might.json [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+      },
+      {
+        "class": "BuffSpell",
+        "attackBonus": 3,
+        "hpBonus": 2
+      }
+    ]
+  },
+  "collectible": true,
+  "set": "RISE_OF_SHADOWS",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/spell_dawns_might.json]: This should be CUSTOM

File: cards/src/main/resources/cards/custom/group3/set_1087/spell_defensive_bearing.json [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+        "hpBonus": 2
+      },
+      {
+        "class": "AddAttributeSpell",
+        "attribute": "TAUNT"
+      }
+    ]
+  },
+  "collectible": true,
+  "set": "RISE_OF_SHADOWS",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/spell_defensive_bearing.json]: This should be CUSTOM

File: cards/src/main/resources/cards/custom/group3/set_1087/spell_dawns_might.json [hunk 16]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Dawn's Might",
+  "baseManaCost": 3,
+  "type": "SPELL",
+  "heroClass": "RUST",
+  "rarity": "COMMON",
+  "description": "Twinspell. Give a minion +3/+2.",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/spell_dawns_might.json]: Replace with

`Give a minion +3/+2. Add one more copy of this to your hand.`

File: cards/src/main/resources/cards/custom/group3/set_1087/twinspell_dawns_might.json [hunk 17]
```diff
@@ -0,0 +1,17 @@
...
+  "rarity": "COMMON",
+  "description": "Give a minion +3/+2.",
+  "targetSelection": "MINIONS",
+  "spell": {
+    "class": "BuffSpell",
+    "attackBonus": 3,
+    "hpBonus": 2
+  },
+  "collectible": false,
+  "set": "RISE_OF_SHADOWS",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/twinspell_dawns_might.json]: This should be CUSTOM

File: cards/src/main/resources/cards/custom/group3/set_1087/weapon_energized_spear.json [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+          "attribute": "MAX_HP"
+        }
+      }
+    }
+  },
+  "attributes": {
+    "SUPREMACY": true
+  },
+  "collectible": true,
+  "set": "RISE_OF_SHADOWS",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group3/set_1087/weapon_energized_spear.json]: This should be CUSTOM

File: cards/src/main/resources/cards/hearthstone/rise_of_shadows/Mage/minion_mana_cyclone.json [hunk 19]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Mana Cyclone",
+  "name": "clone",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/hearthstone/rise_of_shadows/Mage/minion_mana_cyclone.json]: This file was inappropriately renamed, revert this change

File: www/whatsnew.md [hunk 20]
```diff
@@ -3,13 +3,16 @@ layout: page
...
+
+ - 14 new cards in the Dragoon class.
 
 ### 0.8.33-2.0.30 (In Progress)
 
 Bug fixes.
 
 ##### Content Fixes:
-
+ -
```
Comment (doctorpangloss) [inline www/whatsnew.md]: Remove this spurious dash

File: cards/src/main/resources/cards/custom/group3/all_hallows_end/minion_wee_squashling.json (hunk 1 above)
Comment (tranCzhu) [reply cards/src/main/resources/cards/custom/group3/all_hallows_end/minion_wee_squashling.json]: Nope, looks like it's because I hit"enter" in some regex search and it deleted or changed the line
